test/conformance/testdata/conformance.yaml

```diff
@@ -1758,6 +1758,15 @@
     same node if the HostIP or Protocol is different.
   release: v1.16
   file: test/e2e/scheduling/predicates.go
+- testname: Pod preemption verification
+  codename: '[sig-scheduling] SchedulerPreemption [Serial] PreemptionExecutionPath
+    runs ReplicaSets to verify preemption running path [Conformance]'
+  description: Four levels of Pods in ReplicaSets with different levels of Priority,
+    restricted by given CPU limits MUST launch. Priority 1 - 3 Pods MUST spawn first
+    followed by Priority 4 Pod. The ReplicaSets with Replicas MUST contain the expected
+    number of Replicas.
+  release: v1.18
```

Comment:
The bazel test job is failing because you need to update the version here too

Reply:
My mistake, thanks for finding that. Fixed in 63632118b7f